app/ui/design-system/src/lib/Pages/ConceptsPage/index.tsx

```diff
@@ -0,0 +1,150 @@
+import { ButtonLink } from '../../Components/Button';
+import {
+  ContentNavigation,
+  ContentNavigationProps,
+} from '../../Components/ContentNavigation';
+import { EventCard, EventCardProps } from '../../Components/EventCard';
+import {
+  FeatureLinkBlock,
+  FeatureLinkBlockProps,
+} from '../../Components/FeatureLinkBlock';
+import {
+  LandingHeader,
+  LandingHeaderProps,
+} from '../../Components/LandingHeader';
+import { ToolCard, ToolCardProps } from '../../Components/ToolCard';
+
+export interface ConceptsPageProps {
+  landingHeaderItems: LandingHeaderProps;
+  featureLinkBlockItems: [FeatureLinkBlockProps, FeatureLinkBlockProps];
+  toolCardItems: [
+    ToolCardProps,
+    ToolCardProps,
+    ToolCardProps,
+    ToolCardProps,
+    ToolCardProps,
+    ToolCardProps
+  ];
+  contentNavigationItems: [
+    ContentNavigationProps,
+    ContentNavigationProps,
+    ContentNavigationProps
+  ];
+  eventCardItems: EventCardProps;
+}
+
+export function ConceptsPage({
+  landingHeaderItems,
+  featureLinkBlockItems,
+  toolCardItems,
+  contentNavigationItems,
+  eventCardItems,
+}: ConceptsPageProps) {
+  return (
+    <div className="bg-primary-gray-50 dark:bg-black">
+      <div className="pb-10 md:border-b md:border-primary-gray-100 md:pb-28">
```

Comment:
Can the sections be changed to use the PageSections and PageSection components? Check out the Community page for an example